app/scripts/lib/createDupeReqFilterStream.test.ts

```diff
@@ -0,0 +1,315 @@
+import { Transform } from 'stream';
```

Comment:
@danjm Is there any reason why you prefer runtime-specific `stream` over suggested and deterministic `readable-stream` here?

Reply:
oops, no

fixed in d20349a332